extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -333,7 +333,9 @@ export class AmpAnalytics extends AMP.BaseElement {
         `${TAG}: No friendly parent amp-ad element was found for ` +
         'amp-analytics tag with iframe transport.');
 
-    this.iframeTransport_ = new IframeTransport(this.getAmpDoc().win,
+    this.iframeTransport_ = new IframeTransport(
+        // We create a nested cross-domain iframe within the ads if isInabox.
```

Comment:
Nit, change comment to: // Create 3p transport frame within creative frame if inabox